index.html

```diff
@@ -0,0 +1,217 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Timothy | Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
```

Comment:
Great job actualizing the hamburger with the `divs` but I think it will be better if you make use of an icon or image, this will make your code dry. 😊